gists/helm-component-extraction/kubeprometheusstack/scripts/restore_snapshot.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+## Usage
+# ./restore_snapshot.sh <snapshot_directory_name>
+
+SNAPSHOT_DIRECTORY_NAME=$1
+NS="monitoring"
+
+DRY_RUN=client
+
+pausePrometheus() {
+    echo "💀 Turning off the Prometheus instance with replicas: 0"
+    kubectl patch --namespace=${NS} --dry-run=${DRY_RUN} Prometheus kube-prometheus-stack-prometheus --type="json" \
+        --patch '[{"op": "replace", "path": "/spec/replicas", value: 0}]'
+}
+
+resumePrometheus() {
+    echo "🔥 Turning on the Prometheus instance with replicas: 1"
+    kubectl patch --namespace=${NS} --dry-run=${DRY_RUN} Prometheus kube-prometheus-stack-prometheus --type="json" \
+        --patch '[{"op": "replace", "path": "/spec/replicas", value: 1}]'
+}
+
+createDebuggerPod() {
+    echo "🤖 Creating debugger pod"
+    kubectl apply --namespace=${NS} --dry-run=${DRY_RUN} -f templates/debugger.yaml
+
+    echo "🕐 Waiting for debugger pod to be ready"
```

Comment:
Fix your echos, so they hit inside if / else (if not you get duplicate outputs when you dry run)